src/sgame/sg_bot_util.cpp

```diff
@@ -1647,6 +1648,20 @@ void BotFireWeaponAI( gentity_t *self )
 	}
 }
 
+bool BotChangeClass( gentity_t *self, class_t newClass, vec3_t newOrigin )
```

Comment:
```suggestion
static bool BotChangeClass( gentity_t *self, class_t newClass, vec3_t newOrigin )
```
The function is used only in this file

Reply:
still not `static`?